helm/templates/tests/e2etests.yaml

```diff
@@ -6,10 +6,7 @@ metadata:
     "helm.sh/hook": test
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
 spec:
-  template:
-    metadata:
-      labels:
-        admission.datadoghq.com/enabled: "true" # Enable Admission Controller to mutate new pods part of this deployment
+  template: # Enable Admission Controller to mutate new pods part of this deployment
```

Comment:
I'm fine to remove the admission controller label, but we should clean up the comment too.